src/reference/interactive/index.pug

```diff
@@ -254,50 +270,50 @@ block reference
-        li Is this project ready for every Beam broadcaster to have access to?
-        li Is the info tab filled in with #[strong detailed] installation and setup instructions?
+        li Is your project something that everyone would use?
+        li Is your project ready for every Beam broadcaster to have access to?
+        li Have you filled in the Info tab with #[strong detailed] installation and setup instructions?
 
-    p If the answer to any of these is no, then the project should #[strong not] be published. For cases where the project would suit being shared with a smaller group of users, it can be #[a(href='#sharing-your-project') shared] instead.
+    p If you’ve answered no to any of the questions above, you should #[strong not] publish your project yet. If you would like to have a small group of users to be able to access it, consider #[a(href='#sharing-your-project') sharing] your project instead.
 
-    h4#publishing-flow Publishing Flow
+    h4#publishing-flow Understand the Publishing Flow
     .figure
         img(src='/img/reference/interactive/studio/PublishProcess.svg' alt='studio pubish process')
-    p A Project starts off in the draft state. In this state, you can freely make edits to it and test and share it with other users. Many projects don't leave the draft state. It's perfectly ok not to publish your project.
+    p A Project starts off as a draft. In this state, you are free to edit, test, and share it with other users. Many projects don't leave the draft state. It's perfectly ok not to publish your project.
 
-    p Once you've decided to publish your project, visit the publish tab in the interactive studio. It will perform some checks on the project to ensure that it meets some basic requirements before allowing you to submit the project.
-    p Once submitted the Interactive review team at Beam are notified and will begin a review. While a project is in review you won't be able to edit the project. A Beam representative may also reach out to you with questions.
-    p After a review is conducted, your project will either be accepted for publishing or denied and returned the draft state with feedback on what would need to change to get the project to a publishable state.
-    p Should your project be accepted, a date can be discussed with the Beam team and the developers of the project. On this date, the project will be made available for all Beam users in our Interactive Store.
+    p When you've decided to publish your project, go to the Publish tab in the interactive studio. It will perform some checks on the project to ensure that it meets some basic requirements before allowing you to click on the Submit button.
+    p Once you've submitted your project, the Interactive review team at Beam is notified and we'll begin the review process. While a project is in review, you won't be able to edit the project. A Beam representative may also reach out to you with questions.
```

Comment:
we've gone from "Interactive review team at beam" to "we'll" this change goes from slightly formal to slightly informal. Not sure which is best